Dalamud/Plugin/Internal/Profiles/PluginManagementCommandHandler.cs

```diff
@@ -236,12 +236,8 @@ void Continuation(Task t, string onSuccess, string onError)
                     .ConfigureAwait(false);
                 break;
             case PluginCommandOperation.Toggle:
-                this.chat.Print(Loc.Localize("PluginCommandsToggling", "Toggling plugin \"{0}\"...").Format(plugin.Name));
-                Task.Run(() => plugin.State == PluginState.Loaded ? plugin.UnloadAsync() : plugin.LoadAsync(PluginLoadReason.Installer))
-                    .ContinueWith(t => Continuation(t,
-                                      Loc.Localize("PluginCommandsToggleSuccess", "Plugin \"{0}\" toggled.").Format(plugin.Name),
-                                      Loc.Localize("PluginCommandsToggleFailed", "Failed to toggle plugin \"{0}\". Please check the console for errors.").Format(plugin.Name)))
-                    .ConfigureAwait(false);
+                var isDisabling = plugin.State == PluginState.Loaded;
```

Comment:
Sorry for the delay on this, we're prepping API12 things.

Please move this `Toggle` case _before_ the switch statement so we don't recurse and double-call everything above.